### ISSUE TITLE
Hide regions from model explorer and forward to state machines property view

### ISSUE THREAD

**Issue by planger** [eclipsesource/papyrus-umllight#56]
As raised by @tortmayr in #50

> It may be worth eliding the region entirely in both the Model Explorer view (using its facet customization
capability) and in the diagram (presenting the state machine properties when the region is selected) as the definition of multiple regions, and especially composite states, is not supported in UML Light.

**Comment by tortmayr** [comment]
I'm wondering how hiding <i>Region</i> would affect the `New Child` menu for state machines since all other statemachine concepts need a  <i>Region</i> as parent container. I guess I would not be possible to create these concepts via model explorer without further customization

**Comment by cdamus** [comment]
Right.  With the appropriate _edit-helper advice_ (in particular, _configure context_), we can enable creation of vertices in the state machine.  Transitions will not be a problem as they are already supported in the **New Relationship** menu.
